WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTableViewCell.swift

```diff
@@ -53,6 +57,14 @@ final class OrderTableViewCell: UITableViewCell & SearchResultCell {
         paymentStatusLabel.applyStyle(for: viewModel.status)
         paymentStatusLabel.text = viewModel.statusString
 
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.pointOfSaleOrdersi1) {
+            salesChannelLabel.isHidden = false
+            salesChannelLabel.applySalesChannelStyle()
+            salesChannelLabel.text = viewModel.salesChannel
+        } else {
+            salesChannelLabel.isHidden = true
```

Comment:
I'm using `isHidden` here so we don't need to delete the placeholder from the .xib file, otherwise would show up anyway despite being unset, and I prefer to have it display in the .xib editor instead.